src/11_harjoitus_11.Rmd

```diff
@@ -263,10 +263,10 @@ Esikatsele tuloksia painamalla **Apply**.
 Tulos ei ole vielä hyvä, tarvitaan muutama teksteihin tarkoitettu erikoiskomento:
 
 ::: code-box
--gt-label-follow-line: true; -gt-label-max-angle-delta: 90; -gt-label-max-displacement: 400; -gt-label-repeat: 150;
+label-follow-line: true; -gt-label-max-angle-delta: 90; label-max-displacement: 400; -gt-label-repeat: 150;
```

Comment:
@jeeemil Do "-gt-label-max-angle-delta" and "-gt-label-repeat still" include "gt-" or should it be removed here too?